boards/pico/Cargo.toml

```diff
@@ -1,13 +1,15 @@
 [package]
-name = "pico-bsp"
+name = "pico"
```

Comment:
`pico` feels a little too generic if this ever ends up on `crates.io` -- maybe `pico-rp2040`?

Reply:
Maybe `pi-pico`, or `rp-pico` if the idea is to namespace it? `pico-rp2040` feels a bit redundant to me.

Reply:
Yeah I also feel more for prefixing with manufacturer